ui/components/app/transaction-list/transaction-list.component.js

```diff
@@ -123,23 +123,36 @@ export default function TransactionList({
             <div className="transaction-list__header">
               {`${t('queue')} (${pendingTransactions.length})`}
             </div>
-            {pendingTransactions.map((transactionGroup, index) =>
-              transactionGroup.initialTransaction.transactionType ===
-              TransactionType.smart ? (
-                <SmartTransactionListItem
-                  isEarliestNonce={index === 0}
-                  smartTransaction={transactionGroup.initialTransaction}
-                  transactionGroup={transactionGroup}
-                  key={`${transactionGroup.nonce}:${index}`}
-                />
-              ) : (
-                <TransactionListItem
-                  isEarliestNonce={index === 0}
-                  transactionGroup={transactionGroup}
-                  key={`${transactionGroup.nonce}:${index}`}
-                />
-              ),
-            )}
+            {pendingTransactions
+              ///: BEGIN:ONLY_INCLUDE_IN(build-mmi)
+              .sort(
+                (a, b) => b.primaryTransaction.time - a.primaryTransaction.time,
+              )
```

Comment:
Can you describe what this sort is doing so I understand. I'm not sure we don't want this in non MMI :)

Reply:
We added this in order to have/see the pending transactions sorted from the most recent to the oldest, in order for the user to see the most recent on top. It's useful when they more than 2 pending transactions. We've also added it to the completed transactions (we've just forgot to add it here). Also to have the completed transactions sorted by recent -> oldest.